src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -49,7 +49,7 @@ class IqPolicyEvaluatorUtil
               log: loggerBridge))
 
       def verified = iqClient.verifyOrCreateApplication(applicationId)
-      checkArgument(verified, 'Argument iqApplication ' + applicationId + ' failed verification.')
+      checkArgument(verified, 'Argument iqApplication ' + applicationId + ' is invalid.')
```

Comment:
The error message in the other clients is:
"The application ID %s is invalid."

Reply:
> The error message in the other clients is:
"The application ID %s is invalid."

You don't think to call it application ID would be confusing?  I see it being referred to as Argument iqApplication in Jenkins.

Reply:
> You don't think to call it application ID would be confusing? I see it being referred to as Argument iqApplication in Jenkins.

It really is the application ID that is invalid. I don't know why we would call it anything else.

Reply:
> It really is the application ID that is invalid. I don't know why we would call it anything else.

Changed 32b411f